skin.tcl

```diff
@@ -207,6 +207,10 @@ proc iconik_status_tap {} {
 	if {$::de1_num_state($::de1(state)) == "Espresso"} {
 		start_next_step
```

Comment:
I think this now needs a return?

Reply:
Absolutely! Sorry, I just added a commit in this PR to fix it: c320265399e0371dcbf16fdeeed2104c8d35a017